scripts/provisioning/s3_prov_config.yaml

```diff
@@ -104,6 +105,7 @@ TEST:
 DEFAULT_POST_INSTALL:
 DEFAULT_PREPARE:
 DEFAULT_CONFIG:
+  CONFSTORE_SETUP_SIZE: "small"
```

Comment:
we need to confirm about what should be default value for this.

Reply:
Updated to large.